core/new-gui/src/app/workspace/service/dynamic-schema/schema-propagation/schema-propagation.service.ts

```diff
@@ -218,6 +218,16 @@ export class SchemaPropagationService {
       if (v.additionalEnumValue) {
         attrNames.push(v.additionalEnumValue);
       }
+      if (v.title && !operatorSchema.jsonSchema.required?.includes(v.title)) {
+        if (v.default) {
+          if (typeof v.default !== "string") {
+            throw new Error("default value must be a string");
+          }
+          attrNames.push(v.default);
+        } else {
+          attrNames.push("");
+        }
+      }
```

Comment:
Please also document in PR description or code, that the `null` -> `""` change is done by `Ajv.validate()` with [`useDefault`](https://ajv.js.org/guide/modifying-data.html#assigning-defaults) set to true. It is not converted during schema propagation.